easybuild/easyconfigs/p/Pysam/Pysam-0.16.0-GCC-9.3.0.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# 0.9.1.4:
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+# Modified by: Erich Birngruber
+# Gregor Mendel Institute
+# Modified by: Pavel Grochal
+# INUITS
+
+easyblock = 'PythonPackage'
+
+name = 'Pysam'
+version = '0.16.0'
```

Comment:
@FokkeDijkstra as discussed in https://github.com/easybuilders/easybuild-easyconfigs/pull/10823,
would you mind updating to 0.16.0.1?